DigitalLearningSolutions.Web.AutomatedUiTests/DigitalLearningSolutions.Web.AutomatedUiTests.csproj

```diff
@@ -5,6 +5,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AdditionalFiles Remove="C:\Users\Sherif.Olaboye\Source\Repos\DLSV2\StyleCop.json" />

```

Comment:
I'm not sure why this has  been added. It seems to reference a path on your local machine.

Reply:
Can you revert this change. It is not needed.

Reply:
The project will not build and run I can show you if you are less busy